src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/InlineCompletion/InlineCompletionEndPoint.cs

```diff
@@ -178,7 +178,7 @@ public RegistrationExtensionResult GetRegistration()
                 continue;
             }
 
-            using var formattingContext = FormattingContext.Create(request.TextDocument.Uri, document, codeDocument, s_defaultFormattingOptions, _adhocWorkspaceFactory, isFormatOnType: true, automaticallyAddUsings: false);
+            using var formattingContext = FormattingContext.Create(request.TextDocument.Uri, document, codeDocument, s_defaultFormattingOptions, _adhocWorkspaceFactory, isFormatOnType: false, automaticallyAddUsings: false, hostDocumentIndex: 0, triggerCharacter: '\0');
```

Comment:
Why turn format on type to `false` here?

Reply:
The formatting context here is only used for a few helper methods around indentation strings, and there arent any calls to the actual formatting engine. Since this code isn't supplying a trigger character or caret position, and on type formatting won't work without those two bits of info, it felt safer to switch this _just in case_ someone adds more code later that uses the context for something else.

Reply:
Ah interesting. Wont this format the whole document though? Maybe I'm misreading

Reply:
There is no formatting going on, just constructing the context and using it.